docs/getting_started.ipynb

```diff
@@ -3424,7 +3426,8 @@
       "metadata": {},
       "outputs": [],
       "source": [
-        "!pip install llama-stack-client==0.1.0"
+        "!pip install llama-stack-client==0.1.0\n",
```

Comment:
Can we actually avoid adding explicit version information here?

Reply:
We can, but then we can't guarantee the code will work with the latest LS release.